packages/file-explorer/src/WebdavFileStorageTable.ts

```diff
@@ -105,6 +110,20 @@ export class WebdavFileStorageTable implements FileStorageTable {
     }
   }
 
+  async collapseAll(): Promise<void> {
+    const directoryData = await this.fetchData();
+    const { items } = directoryData;
+    for (let i = 0; i < items.length; i += 1) {
+      if (
+        isDirectory(items[i]) &&
+        !items[i].filename.includes('/', 1) &&
+        (items[i] as DirectoryStorageItem).isExpanded
+      ) {
+        this.setExpanded(items[i].filename, false);
+      }
+    }
```

Comment:
There's an easier way you can do this because of how the expanded tables are handled in `WebdavFileStorageTable`.
The way `WebdavFileStorageTable` works, is it builds a tree of `WebdavFileStorageTable`... each `WebdavFileStorageTable` is just the items in one directory, and then it has a map of `childTables` that are used for each doc. So all you need to do, is just clear out the map of `childTables`, and that removes all directories that are expanded. Something like:
```suggestion
    this.childTables.clear();
    if (this.currentViewport()) {
      await this.refreshInternal();
    }
```